docs/deployment.md

```diff
@@ -0,0 +1,58 @@
+This repository includes code for generating a self-zip archive
+of the code and all of it's Python dependencies. We use these
+archives to deploy the site to a Linux server.
+
+# What's an artifact?
```

Comment:
I propose moving this section down below the "Generating" section, as, at first read, this provides an awful lot of detail without first getting a sense for the purpose of the artifact at a high level.

Reply:
This is done in the latest commit